src/calculateRentalCost.js

```diff
@@ -1,10 +1,19 @@
 /**
- * @param {number} days
- *
- * @return {number}
+ * Calculates the rental cost based on the number of days.
+ * @param {number} days - The number of days the car is rented.
+ * @return {number} - The total rental cost.
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
```

Comment:
Consider using constants for the cost per day and the discount values to avoid hardcoding. This will make the code more readable and maintainable.